content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -28,6 +28,26 @@ located above the project table.
 
 ![](https://static.iterative.ai/img/studio/view_components_1.gif)
 
+### Nested branches
+
+When a Git branch is merged into another branch (e.g., `main`), two
+possibilities exist:
+
+- The user continues to push more commits to the merged branch, which means that
+  the merged branch contains some commits that are not present in `main`. In
+  this scenario, the project table will display both the `main` branch and the
+  merged branch.
```

Comment:
Maybe note that the merged branch will only show the commits that are not present in `main`?